### DISCUSSION TITLE
Event

### DISCUSSION THREAD

**Discussion by Angelk90** [community/community#16762] [Mobile] [answered]
On the Github home page there are the following two tabs, which allow you to see the events:

Following:
- of the people you follow (started following, starred, fork, create release, created a repository, etc..)
- of yourself (if a person put a star in your project, if a person forked your project, etc..)

For you:
Similar to `Following`:
- only that it allows you to see events too, with people you interacted with but don't follow.
- the projects that you `Recommended for you`

<img width="239" alt="Schermata 2022-05-17 alle 10 56 59" src="https://user-images.githubusercontent.com/20476002/168772262-a974c751-034b-46c8-908b-5eac3eb3d15f.png">

**Comment by mxie** [comment, marked as answer]
Hi @Angelk90 - is there a question about how this works?

**Reply by Angelk90** [reply]
Hi @mxie - What do you mean, how does it work?

**Reply by mxie** [reply]
Can you clarify what you're requesting or reporting feedback on? Is there something missing that you'd like to see?

**Reply by Angelk90** [reply]
@mxie , In the github homepage, there is the sequences section, as seen from the video it allows you to see the following information, the elecante at the beginning.

What I would like to see in the app are the following two sections, `Following` and `For you`.

https://user-images.githubusercontent.com/20476002/168898506-abfd3cb5-a961-475f-a5be-42c87a48980e.mov

**Reply by mxie** [reply]
Ahhh ok, thank you for elaborating! We're actually in the process of building this out. Stay tuned for its release in a few weeks.

**Reply by Angelk90** [reply]
@mxie: Ok, thanks.
I wait to see the results.

**Reply by mxie** [reply]
Hi @Angelk90 - I just wanted to follow up on this and let you know that we had already rolled out changes to the Explore tab to match the `For you` section.

**Reply by Angelk90** [reply]
Hi @mxie , but the Following section is not there?

**Reply by Angelk90** [reply]
@mxie , I noticed a problem if for example, one of the users followed several people at the same time or put a star on several projects in the same period of time.

In the online version it shows it, grouping them together using the "Show all" and "Show less" buttons.

In the android app version, however, no, it does not show them together or separate.

But it shows only one, the first on the list.

**Reply by Angelk90** [reply]
@mxie : Another problem as you can see is that it does not show the preview video in explore, it puts the http address.
To see the video I have to click on "Read more".
![image](https://user-images.githubusercontent.com/20476002/181836024-52c1d6e4-b64d-40d7-8235-1e50d1402237.png)

**Reply by Angelk90** [reply]
@mxie : I don't even see in explore if a user has sponsored a project, but in the online version it shows.
